source/retryable-reads/tests/README.rst

```diff
@@ -0,0 +1,269 @@
+=====================
+Retryable Reads Tests
+=====================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to prove their conformance to the Retryable Reads spec.
+
+Prose tests, which are not easily expressed in YAML, are also presented
+in this file. Those tests will need to be manually implemented by each driver.
+
+Tests will require a MongoClient created with options defined in the tests.
+Integration tests will require a running MongoDB cluster with server versions
+4.2.0 or later. N.B. Strictly speaking, the server should implement the
+"Early Failure on Socket Disconnect" slated for 4.2.0, but this is not
+neccessary for initial testing.
+
+
+Server Fail Point
+=================
+
+Some tests depend on a server fail point, expressed in the ``failPoint`` field.
+For example the ``failCommand`` fail point allows the client to force the
+server to return an error. Keep in mind that the fail point only triggers for
+commands listed in the "failCommands" field. See `SERVER-35004`_ and
+`SERVER-35083`_ for more information.
+
+.. _SERVER-35004: https://jira.mongodb.org/browse/SERVER-35004
+.. _SERVER-35083: https://jira.mongodb.org/browse/SERVER-35083
+
+The ``failCommand`` fail point may be configured like so::
+
+    db.adminCommand({
+        configureFailPoint: "failCommand",
+        mode: <string|document>,
+        data: {
+          failCommands: ["commandName", "commandName2"],
+          closeConnection: <true|false>,
+          errorCode: <Number>,
+        }
+    });
+Some tests depend on a server fail point, ``failCommand``, which
+allows us to force a network error before the server would return a read result
+to the client.  See `SERVER-35004`_ for
+more information.
+
+.. _SERVER-29606: https://jira.mongodb.org/browse/SERVER-35004
+
+The fail point may be configured like so::
+
+    db.runCommand({
+        configureFailPoint: "failCommand",
+        mode: <string|document>,
+        data: <document>
+    });
```

Comment:
Isn't this redundant with the previous example, which was also for `failCommand` but illustrated the format of the `data` field?